test/Erc1155Quest.spec.ts

```diff
@@ -41,8 +43,8 @@ describe('Erc1155Quest', () => {
     thirdAddress = local_thirdAddress
     fourthAddress = local_fourthAddress
 
-    expiryDate = Math.floor(Date.now() / 1000) + 10000
-    startDate = Math.floor(Date.now() / 1000) + 1000
+    expiryDate = latestTime + 1000
+    startDate = latestTime + 10
```

Comment:
Much more readable! NIce

Reply:
Oh what the real benefit of this is that you don't need to reverse the time. NICE